test/NanoTimeTest.java

```diff
@@ -0,0 +1,15 @@
+
+import static avian.testing.Asserts.*;
+
+public class NanoTimeTest {
+
+  public static void main(String[] args) throws InterruptedException {
+    long last = System.nanoTime();
+    for(int i=0; i<100; i++) {
+      Thread.sleep(1);
+      long nt = System.nanoTime();
+      assertTrue(nt>last);
```

Comment:
It is possible this test could fail on non linux platforms.  It shouldnt, but it has a higher chance of it.